packages/framework/src/Resources/views/Admin/Content/SalesRepresentative/edit.html.twig

```diff
@@ -0,0 +1,4 @@
+{% extends '@ShopsysFramework/Admin/Content/SalesRepresentative/detail.html.twig' %}
+
+{% block title %}- {{ 'Editing sales representative'|trans }} - {{ salesRepresentative.firstName }} {{ salesRepresentative.lastName }}{% endblock %}
+{% block h1 %}{{ 'Editing sales representative'|trans }} - {{ salesRepresentative.firstName }} {{ salesRepresentative.lastName }}{% endblock %}
```

Comment:
these can also be replaced with the `fullName` method

Reply:
done